addon/components/one-way-input.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember';
 
```

Comment:
I shall kill it for you! I figured it was desired from here https://github.com/DockYard/ember-one-way-controls/blob/master/addon/components/one-way-select.js#L3